app/assets/stylesheets/variables.scss

```diff
@@ -10,12 +10,9 @@ $floral-white: #f9f6ef;
 $tahuna-sands: #d2c295;
 
 // Accordion
-$accordion-button-color: $white;
-$accordion-button-bg: $sonic-silver;
-$accordion-button-active-color: $white;
-$accordion-button-active-bg: $sonic-silver;
-$accordion-button-focus-box-shadow: 0 0 0.25rem 0.25rem
-  rgba($sonic-silver, 0.25);
+$accordion-button-bg: var(--stanford-10-black);;
```

Comment:
Can you set `--bs-accordion-btn-bg` and `--bs-accordion-active-bg` instead on `.document-component`?

Reply:
```
.document-component {
  --bs-accordion-btn-bg: var(--stanford-10-black);
  --bs-accordion-active-bg: var(--stanford-10-black);
}
```
doesn't work. Do I have this rights?